clients/java/src/main/java/io/zeebe/client/impl/util/PropertyUtils.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.client.impl.util;
+
+public final class PropertyUtils {
```

Comment:
Please make this class package private so that users of the client don't have direct access to this class (to do so, you also need to move it into `io.zeebe.client.impl` so that it can be accessed by the builders). I'd also prefer to rename this class to `BuilderUtils` or so.

Reply:
Done